<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pcl_visualizer</class>
 <widget class="QMainWindow" name="pcl_visualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pcl_visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pcl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>open_pcd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pic">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>open_pic</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelimage">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>36</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
